src/Relative.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Box from './Box'
+import { responsiveStyle, style } from 'styled-system'
+
+const top = responsiveStyle({
+  prop: 'top',
+  cssProperty: 'top',
+  numberToPx: true
+})
+
+const bottom = responsiveStyle({
+  prop: 'bottom',
+  cssProperty: 'bottom',
+  numberToPx: true
+})
+
+const left = responsiveStyle({
+  prop: 'left',
+  cssProperty: 'left',
+  numberToPx: true
+})
+
+const right = responsiveStyle({
+  prop: 'right',
+  cssProperty: 'right',
+  numberToPx: true
+})
+
+const zIndex = style({
+  prop: 'zIndex',
+  cssProperty: 'z-index',
+  numberToPx: false
+})
+
+const Relative = styled(Box)`
+  position: relative;
+  ${top} ${bottom} ${left} ${right}
+  ${zIndex}
+`
+
+const numberStringOrArray = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.string,
+  PropTypes.array
+])
+
+Relative.propTypes = {
+  top: numberStringOrArray,
+  bottom: numberStringOrArray,
+  left: numberStringOrArray,
+  right: numberStringOrArray,
```

Comment:
See that these position props accept one of types from `numberStringOrArray`, but your story only provides examples of the number type. It'd be great if the story can provide some examples of the String or Array type, and the doc of `Relative` can provide a brief emphasis what expected CSS rule(s) for each of these types used in the position props?

Reply:
Good point.  We use `numberStringOrArray` all over design-system, but we're not explaining it.

Let's discuss how we want to convey this information across all pages and make it a separate issue.